src/Medology/Behat/CsvContext.php

```diff
@@ -118,4 +119,28 @@ private function assertNoAdditionalHeaders(array $required, array $actual, strin
             throw new Exception("CSV '$csvName' contains extra headers '" . implode("', '", $diff) . "'");
         }
     }
+
+    /**
+     * Update expected data injecting model values.
+     *
+     * @param TableNode $table a table node containing the values to inject
```

Comment:
The table does not contain values to inject. Values are being injected into it.

```suggestion
     * @param TableNode $table a table node containing the values to parse
```